plugins/package-managers/node/src/funTest/kotlin/pnpm/PnpmFunTest.kt

```diff
@@ -66,7 +67,7 @@ class PnpmFunTest : StringSpec({
         val definitionFile = getAssetFile("projects/synthetic/pnpm/workspaces/packages.json")
         val expectedResultFile = getAssetFile("projects/synthetic/pnpm/workspaces-expected-output.yml")
 
-        val result = analyze(definitionFile.parentFile, packageManagers = setOf(PnpmFactory()))
+        val result = analyze(definitionFile.parentFile, packageManagers = setOf(PnpmFactory())).getAnalyzerResult()
```

Comment:
I propose instead:

```kotlin
val ortResult = analyze(definitionFile.parentFile, packageManagers = setOf(PnpmFactory()))

ortResult.analyzer shouldNotBeNull {
    patchActualResult(result.toYaml(), patchStartAndEndTime = true) should
        matchExpectedResult(expectedResultFile, definitionFile)
}
```

Note the renaming of `result` to `ortResult` so that `result.toYaml()` can be used. Alternatively, explicitly use `this.result.toYaml()` and keep the `result` variable name.

Reply:
I like the approach as-is more, because it is less verbose. Would it be ok to keep as-is?

(Why do you prefer the above?)

Reply:
> Why do you prefer the above?

I prefer my proposal as it guards all assumptions being made with test assertions. 

While it's correct that the test would fail in any case if `analyzer == null`, `shouldNotBeNull` makes more clear than `checkNotNull` that this should not happen in the context of *this test*.

But this probably is a matter of taste anyway, and as you're making reuse of the introduced extension function as part of the next commit, I'm fine with keeping it as-is.